packages/ia-components/.babelrc

```diff
@@ -1,22 +1,23 @@
 {
-    "presets": [
-        [
-          "@babel/env", {
-            "targets": {
-              "node": "current"
-            }
-          }
-        ],
-        "@babel/preset-react"
+  "presets": [
+    [
+      "@babel/env",
+      {
```

Comment:
was there any issue with the linting in the previous version? not sure if we drop the first bracket down to the next line or not.

Reply:
also, make sure @babel/env (or preset-env? which is the latest one to use?) is loaded into package.json

Reply:
@babel/env is shorthand for @babel/preset-env. Both are acceptable as per Babel docs.

Reply:
> was there any issue with the linting in the previous version? not sure if we drop the first bracket down to the next line or not.

The changes done was the default behaviour of the "Babel upgrade" package used by me. But I think it is perfectly fine to ignore this linting change in this PR.

Reply:
> also, make sure @babel/env (or preset-env? which is the latest one to use?) is loaded into package.json

@iisa yes, the presets were missing in the `package.json`. I have updated the PR with the changes.